python/paddle/fluid/tests/test_cpp_reader.py

```diff
@@ -15,16 +15,30 @@
 import paddle.v2 as paddle
 import paddle.fluid as fluid
 import numpy as np
+import sys
 
-prog = fluid.framework.Program()
-block = prog.current_block()
+startup_prog = fluid.framework.Program()
```

Comment:
Why make the reader operator as a part of startup program?

Reply:
Because a reader should only be created once.